examples/mlflow-3/sklearn_autolog.py

```diff
@@ -0,0 +1,24 @@
+# ruff: noqa
```

Comment:
@harupy Can you add a similar example that uses GridSearch and make sure that the outputs make sense (e.g. correct loggedmodels are produced, linked to the correct runs, and have the correct metrics?)

Reply:
Added!